gui/ged/inc/TPaveStatsEditor.h

```diff
@@ -28,23 +21,23 @@ class TPaveStats;
 class TPaveStatsEditor : public TGedFrame {
 
 protected:
-   TPaveStats         *fPaveStats;        // TPaveStats object
-   // widgets for stat options
-   TGCheckButton      *fHistoName;        // histo name check box
-   TGCheckButton      *fEntries;          // entries' number check box
-   TGCheckButton      *fMean;             // mean value check box
-   TGCheckButton      *fRMS;              // RMS check box
-   TGCheckButton      *fUnderflow;        // underflow number check box
-   TGCheckButton      *fOverflow;         // overflow number check box
-   TGCheckButton      *fIntegral;         // integral of bins check box
-   TGCheckButton      *fSkewness;         // skewness check box
-   TGCheckButton      *fKurtosis;         // kurtosis check box
-   TGCheckButton      *fStatsErrors;      // statistics error check box
-   // widgets for fit options
-   TGCheckButton      *fNameValues;       // parameters' name/values check box
-   TGCheckButton      *fErrors;           // error check box
-   TGCheckButton      *fChisquare;        // Chisquare check box
-   TGCheckButton      *fProbability;      // probability check box
+   TPaveStats         *fPaveStats;        ///< TPaveStats object
+   ///< widgets for stat options
+   TGCheckButton      *fHistoName;        ///< histo name check box
+   TGCheckButton      *fEntries;          ///< entries' number check box
+   TGCheckButton      *fMean;             ///< mean value check box
+   TGCheckButton      *fRMS;              ///< RMS check box
+   TGCheckButton      *fUnderflow;        ///< underflow number check box
+   TGCheckButton      *fOverflow;         ///< overflow number check box
+   TGCheckButton      *fIntegral;         ///< integral of bins check box
+   TGCheckButton      *fSkewness;         ///< skewness check box
+   TGCheckButton      *fKurtosis;         ///< kurtosis check box
+   TGCheckButton      *fStatsErrors;      ///< statistics error check box
+   ///< widgets for fit options
+   TGCheckButton      *fNameValues;       ///< parameters' name/values check box
```

Comment:
There are very few such cases, may be it is the only file.

Reply:
I will cross check once more.